### DISCUSSION TITLE
Can I use f1-dash as is in a live stream on Youtube?

### DISCUSSION THREAD

**Discussion by ijro** [slowlydev/f1-dash#162] [General]
I do some live streaming on YouTube and would like to use f1-dash as an extra screen to entertain the audience while I do race commentary.
Would there be any restrictions from the team that develops or owns f1-dash?

**Comment by slowlydev** [comment]
For now I have no problem of you showing it on your stream. But would like you to give some credit, like adding the link to f1-dash in the streams description
